paper-slider.html

```diff
@@ -375,12 +407,23 @@
     },
 
     _knobdown: function(event) {
-      event.detail.sourceEvent.preventDefault();
+
+      // blur this slider if the user holds down the knob
+      // since we are calling `preventDefault` at the end of this function,
+      // the browser won't do it automatically.
+
       this._expandKnob();
+
+      // set the focus manually because we will call prevent default at the end
+      this.focus();
```

Comment:
It seems like this should happen _after_ the `preventDefault`. Is it better for it to be before?